lib/cli-engine.js

```diff
@@ -670,15 +674,28 @@ class CLIEngine {
             // replace \ with / for Windows compatibility
             format = format.replace(/\\/g, "/");
 
+            let cwd = this.options ? this.options.cwd : process.cwd();
+
+            const namespace = naming.getNamespaceFromTerm(format);
+
+            if (namespace) {
+                cwd = `${cwd}/${namespace}`;
```

Comment:
What is this line doing, can you please explain the with a use case?

Reply:
Please see comment I left above regarding the same.